profiles/schema.py

```diff
@@ -1490,6 +1493,10 @@ def mutate_and_get_payload(cls, root, info, **input):
                 _("You do not have permission to perform this action.")
             )
 
+        if not requester_has_sufficient_loa_to_perform_gdpr_request(info.context):
+            raise PermissionDenied(
```

Comment:
Instead of raising `PermissionDenied` we should raise a new error that our `GraphQLView` is able to render into an error that the UI can use to render a better error message i.e. user need to know that he/she should raise his/her `loa`.